libraries-5/src/test/java/com/baeldung/githubapi/ClientUnitTest.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.githubapi;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.GitHubBuilder;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ClientUnitTest {
```

Comment:
If this is making a live connection it should be a `LiveTest` so it doesn't make the CI build flaky on connection issues